nestkernel/nest_time.cpp

```diff
@@ -162,7 +165,7 @@ Time::fromstamp( Time::ms_stamp t )
   // intended ones.
   tic_t n = static_cast< tic_t >( t.t * Range::TICS_PER_MS );
   n -= ( n % Range::TICS_PER_STEP );
-  long s = n / Range::TICS_PER_STEP;
+  long s = n * Range::TICS_PER_STEP_INV;
   double ms = s * Range::MS_PER_STEP;
```

Comment:
What we need here is just MS_PER_TIC (MS_PER_STEP * STEP_PER_TIC).

This still seems like an overcomplex as per the comment on lines 160 + 161, a computation with ceil should be sufficient, but there's corner cases I haven't checked.